src/GitHub.Exports/Settings/PkgCmdID.cs

```diff
@@ -18,5 +19,10 @@ public static class PkgCmdIDList
         public const int githubCommand = 0x320;
         public const int helpCommand = 0x321;
         public const int blameCommand = 0x500;
```

Comment:
Arg this isn't the line I thought I commented on.  I tried to comment on the following line:
```
        public const int openPullRequestsCommand = 0x201;
```